lib/commands/types.ts

```diff
@@ -191,7 +191,7 @@ export interface WebviewsMapping {
   /**
    * Webview pages list as it is retrieved by `/json/list` CDP endpoint
    */
-  pages: StringRecord[];
+  pages?: StringRecord[];
```

Comment:
Because of these lines, this can be `undefined`.

https://github.com/appium/appium-android-driver/blob/master/lib/helpers/webview.ts#L330-L335

Reply:
Please add this comment into the docstring

Reply:
How about something like this d0057995693acb4bd35aa124890593e06940f10d?